### ISSUE TITLE
Cheat Detection is not working as expected

### ISSUE THREAD

**Issue by AlexZaccaria** [dmustanger/7dtd-ServerTools#450]
Fly mode is banning users when they swim;
Infinite Ammo is banning users when they leave the vehicles (confirmed on bicycles);
Block damage is banning users when they harvest corpses or they remove the claim blocks;

Can you please take a look at this?
Thanks!

**Comment by ObsComp** [comment]
Yes, I will take a peak. There is a planned full redo of the cheat system. I needed to get the core fundamentals of ServerTools working proper again which they are thankfully now, so I can start to work on the anti cheat very soon. I would not trust any of it right now. Most of it was developed, coding and designed for a19. It has not been touched in that long. Thank you for the report.
